source/exe/win32/platform_checks.cc

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "envoy/filesystem/filesystem.h"
+
+namespace Envoy {
+
+// Check system wide configurations such as user level
+// file watches, file limit permissions etc...
+// TODO(windows maintainers)
```

Comment:
@wrowe for visibility. I think you should probably put a bug # here. No one will know to search for that particular string (whereas sometimes I might grep the codebase for TODO(jmarantz)).

But probably a better pattern is TODO(#xxxx) where xxxx is some issue you add to the issue-tracker for this, and then it can be assigned to whoever.

Reply:
Sorry I Missed that! - Ill create a issue for this rightaway.

Reply:
Created: https://github.com/envoyproxy/envoy/issues/9025